src/pages/setting/index.jsx

```diff
@@ -12,48 +12,62 @@ const Setting = () => {
 
   const appArr = [
     {
-      label: "앱 소개",
+      label: "공지사항",
       style: S.HoverPlain,
       onClickHandler: () => {
-        console.log("노션 페이지로 리디렉션");
+        window.open(
+          "https://ecopercent.notion.site/ecopercent/eco-3932cd03b235420197d6c81094e4b0e1",
+          "_blank"
+        );
       },
     },
     {
-      label: "개발팀",
+      label: "Q & A",
       style: S.HoverPlain,
       onClickHandler: () => {
-        console.log("노션 페이지로 리디렉션");
+        window.open(
+          "https://ecopercent.notion.site/ecopercent/eco-3932cd03b235420197d6c81094e4b0e1",
+          "_blank"
+        );
       },
     },
     {
-      label: "도움말",
-      style: S.HoverPlain,
-      onClickHandler: () => {
-        console.log("노션 페이지로 리디렉션");
-      },
+      label: "문의처",
+      style: S.Plain,
+      additionalLabel: "ecopercent@gmail.com",
+      onClickHandler: () => {},
```

Comment:
```suggestion
      style: S.HoverPlain,
      onClickHandler: () => {
        window.open(
          "mailto:ecopercent@gmail.com", "_blank"
        );
      },
```

mailto 링크로 연결하는게 자연스러울 것 같은데 어떻게 생각하시나요~~?